pmm-admin.go

```diff
@@ -31,18 +31,33 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	ctx    context.Context
+	cancel context.CancelFunc
+)
+
 var (
 	admin pmm.Admin
 
 	rootCmd = &cobra.Command{
 		Use: "pmm-admin",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			// This function pre-runs with every command or sub-command.
-			// Two exceptions are "pmm-admin config" and "pmm-admin uninstall" which bypass it.
+			ctx, cancel = context.WithTimeout(context.Background(), flagTimeout)
 
-			// Skip pre-run for "help" command.
-			// You should always be able to get help even if pmm is not configured yet.
-			if cmd.Name() == "help" {
+			switch cmd.Name() {
+			case "help":
+				// Skip pre-run for "help" command.
+				// You should always be able to get help even if pmm is not configured yet.
+				return
```

Comment:
this return statement may be reached without using the cancel var defined on line 45